### PR TITLE
Added info about file compression to filesystem docs

### PR DESCRIPTION
**Pull request by dat-a-man**
### Description
The filesystem destination within the dlt library employs gzip compression by default to enhance efficiency and reduce storage space requirements. This compression means files are stored in a format that is not immediately readable as plain text or JSON Lines (jsonl) files, potentially complicating direct file access or analysis.

### Additional Context
This update aims to provide clearer information on handling file compression within the dlt library's filesystem destination, ensuring users can efficiently manage and access their data as needed.

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md [hunk 1]
```diff
@@ -185,6 +185,18 @@ bucket_url = "file:///absolute/path"  # three / for absolute path
 - `replace` - all files that belong to such tables are deleted from dataset folder, and then the current set of files is added.
 - `merge` - falls back to `append`
 
+## File Compression
+
+The filesystem destination in the dlt library uses gzip compression by default for efficiency, which may result in the files being stored in a compressed format. This format may not be easily readable as plain text or JSON Lines (jsonl) files. If you encounter files that seem unreadable, they may be compressed.
+
+To handle compressed files:
+
+- To disable compression, you can modify the `data_writer.disable_compression` setting in your "config.toml" file. This can be useful if you want to access the files directly without needing to decompress them.
```
Comment (VioletM) [inline docs/website/docs/dlt-ecosystem/destinations/filesystem.md:194]: Maybe add a code snippet?
```
[normalize.data_writer]
disable_compression=false
```

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md [hunk 2]
```diff
@@ -185,6 +185,18 @@ bucket_url = "file:///absolute/path"  # three / for absolute path
 - `replace` - all files that belong to such tables are deleted from dataset folder, and then the current set of files is added.
 - `merge` - falls back to `append`
 
+## File Compression
+
+The filesystem destination in the dlt library uses gzip compression by default for efficiency, which may result in the files being stored in a compressed format. This format may not be easily readable as plain text or JSON Lines (jsonl) files. If you encounter files that seem unreadable, they may be compressed.
```
Comment (VioletM) [inline docs/website/docs/dlt-ecosystem/destinations/filesystem.md:190]:
```suggestion
The filesystem destination in the dlt library uses `gzip` compression by default for efficiency, which may result in the files being stored in a compressed format. This format may not be easily readable as plain text or JSON Lines (`jsonl`) files. If you encounter files that seem unreadable, they may be compressed.
```

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md (hunk 1 above)
Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/destinations/filesystem.md:194]: Added

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md [hunk 3]
```diff
@@ -185,6 +185,18 @@ bucket_url = "file:///absolute/path"  # three / for absolute path
...
+
+To handle compressed files:
+
+- To disable compression, you can modify the `data_writer.disable_compression` setting in your "config.toml" file. This can be useful if you want to access the files directly without needing to decompress them.
+
+- To decompress a gzip file, you can use tools like gunzip. This will convert the compressed file back to its original format, making it readable.
```
[the remark this answers is not in the document]
Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/destinations/filesystem.md:196]: highlighted

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md (hunk 2 above)
Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/destinations/filesystem.md:190]: highlighted

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md (hunk 1 above)
Comment (VioletM) [reply docs/website/docs/dlt-ecosystem/destinations/filesystem.md]: Oh, sorry, my fault, should be

```
[normalize.data_writer]
disable_compression=true
```

Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/destinations/filesystem.md]: okay, done.

**Review by VioletM — APPROVED** [review]

**Pull request merged**